packages/plugin-legacy/README.md

```diff
@@ -117,6 +117,13 @@ npm add -D terser
 
   Defaults to `false`. Enabling this option will exclude `systemjs/dist/s.min.js` inside polyfills-legacy chunk.
 
+### `renderModernChunks`
+
+- **Type:** `boolean`
+- **Default:** `true`
+
+  Set to `true` to only output the legacy bundles that support all target browsers.
```

Comment:
This seems to be a mistake and should be set to `false` to disable it.

Reply:
Thanks!